orca-sql/src/main/kotlin/com/netflix/spinnaker/orca/sql/cleanup/OldPipelineCleanupPollingNotificationAgent.kt

```diff
@@ -98,37 +98,47 @@ class OldPipelineCleanupPollingNotificationAgent(
   private fun performCleanup() {
     val thresholdMillis = Instant.ofEpochMilli(clock.millis()).minus(thresholdDays, ChronoUnit.DAYS).toEpochMilli()
 
-    val pipelineConfigsWithOldExecutions = jooq
-      .select(field("application"), field("config_id"), count(field("id")).`as`("count"))
+    val candidateApplications = jooq
+      .select(field("application"))
       .from(table("pipelines"))
-      .where(
-        field("build_time").le(thresholdMillis).and(
-          "status IN (${completedStatuses.joinToString(",") { "'$it'" }})"
-        )
-      )
-      .groupBy(field("application"), field("config_id"))
+      .groupBy(field("application"))
```

Comment:
No longer need the `groupBy` at all here.

Reply:
but how would i get the aggregate count?

testing these two queries:
`select application from pipelines group by application having count(id) > 200;`
and
`select application from pipelines having count(id) > 200;`

they don't produce the same result. Or am I misunderstanding what you mean?